src/test/cli/radosgw-admin/help.t

```diff
@@ -287,7 +287,7 @@
      --path-prefix             path prefix for filtering roles
   
     --conf/-c FILE    read configuration from the given configuration file
-    --id/-i ID        set ID portion of my name
+    --id ID           set ID portion of my name
```

Comment:
rgw_admin calls `generic_client_usage()`, which is `generic_usage(false)`. So as per my analysis changing this line to 

`--id              set ID portion of my name` 

as written in `src/common/ceph_argparse.cc` fixes it.

Reply:
I forgot to add the `ID` parameter to the config help. What do yo uthink now?